### PR TITLE
KO: Translations for ko/beta/tutorials/eager

### PR DESCRIPTION
**Pull request by Choiuijin1125**
Translations for ko/alpah/tutorials/eager.
These files were translated based on en/r2/tutorials/eager(https://github.com/tensorflow/docs/tree/master/site/en/r2/tutorials/eager) and
ko/tutorials/eager(https://github.com/tensorflow/docs/tree/master/site/ko/tutorials/eager).

I'll upload the rest file(tf.function.jpynb) sometime soon :) 
Thanks !

### CODE REVIEW

**Comment by Choiuijin1125** [comment]
@lamberta Ohhhh, I was trying to move to beta, But you did!! 
I really appreciate it. thanks, @lamberta Have a good day. :)

**Comment by cre8tor** [comment]
I'll review file by file.

File: site/ko/beta/tutorials/eager/custom_training_walkthrough.ipynb [hunk 1]
```diff
@@ -0,0 +1,1156 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "1J3AuPBT9gyR"
+   },
+   "source": [
+    "### 임포트 구성\n",
+    "\n",
+    "텐서플로와 필요한 파이썬 모듈을 임포트합니다. 텐서플로우는 기본적으로, 텐서플로우는 연산이 나중에 실행되는 [계산 그래프(computational graph)](https://www.tensorflow.org/guide/graphs)를 만드는 대신에 연산을 즉시 평가하고 구체적인 값을 반환하는 [즉시 실행](https://www.tensorflow.org/guide/eager)을 사용합니다. 만약 파이썬 대화형 창이나 상호작용 콘솔을 사용하시면 더욱 익숙할 겁니다. "
```
Comment (cre8tor) [inline site/ko/beta/tutorials/eager/custom_training_walkthrough.ipynb]: [계산 그래프(computational graph)] -> [계산 그래프](computational graph) ?
I can't remember clearly, but I saw a review that don't include English description for the link. But I'm not sure..
This part need review of another reviewer.

File: site/ko/beta/tutorials/eager/custom_training_walkthrough.ipynb [hunk 2]
```diff
@@ -0,0 +1,1156 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "RaKp8aEjKX6B"
+   },
+   "source": [
+    "### 손실 함수와 그래디언트 함수 정의하기\n",
+    "\n",
+    "훈련과 평가단계에서 모델의 *[손실(loss)](https://developers.google.com/machine-learning/crash-course/glossary#loss)*을 계산해야 합니다. 손실은 모델의 예측이 원하는 레이블과 얼마나 일치하는지, 또한 모델이 잘 작동하는지에 대한 척도로 사용됩니다. 이 값을 최소화하고, 최적화 해야합니다.\n",
```
Comment (cre8tor) [inline site/ko/beta/tutorials/eager/custom_training_walkthrough.ipynb]: Italic as above :)

File: site/ko/beta/tutorials/eager/custom_training_walkthrough.ipynb [hunk 3]
```diff
@@ -0,0 +1,1156 @@
...
+   "outputs": [],
+   "source": [
+    "def loss(model, x, y):\n",
+    "  y_ = model(x)\n",
+    "\n",
+    "  return loss_object(y_true=y, y_pred=y_)\n",
+    "\n",
+    "\n",
+    "l = loss(model, features, labels)\n",
+    "print(\"Loss test: {}\".format(l))"
```
Comment (cre8tor) [inline site/ko/beta/tutorials/eager/custom_training_walkthrough.ipynb]: Need translation for 'Loss test'

File: site/ko/beta/tutorials/eager/custom_training_walkthrough.ipynb [hunk 4]
```diff
@@ -0,0 +1,1156 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "3IcPqA24QM6B"
+   },
+   "source": [
+    "모델을 최적화하기 위해 사용되는 *[그래디언트(gradient)](https://developers.google.com/machine-learning/crash-course/glossary#gradient)*를 계산하기 위해 [tf.GradientTape](https://www.tensorflow.org/api_docs/python/tf/GradientTape) 컨텍스트를 사용합니다."
```
Comment (cre8tor) [inline site/ko/beta/tutorials/eager/custom_training_walkthrough.ipynb]: Italic as above

File: site/ko/beta/tutorials/eager/custom_training_walkthrough.ipynb [hunk 5]
```diff
@@ -0,0 +1,1156 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "lOxFimtlKruu"
+   },
+   "source": [
+    "### 옵티마이저 생성 \n",
+    "\n",
+    "*[옵티마이저(optimizer)](https://developers.google.com/machine-learning/crash-course/glossary#optimizer)*는 손실 함수를 최소화하기 위해 계산된 그래디언트를 모델의 변수에 적용합니다. 손실 함수를 구부러진 곡선의 표면(그림 3)으로 생각할 수 있으며, 이 함수의 최저점을 찾고자 합니다. 그래디언트는 가장 가파른 상승 방향을 가리키며 따라서 반대 방향으로 이동하는 여행을 합니다. 각 배치마다의 손실과 기울기를 반복적으로 계산하여 훈련과정 동안 모델을 조정합니다. 점진적으로, 모델은 손실을 최소화하기 위해 가중치(weight)와 편향(bias)의 최적의 조합을 찾아냅니다. 손실이 낮을수록 더 좋은 모델의 예측을 기대할 수 있습니다.\n",
```
Comment (cre8tor) [inline site/ko/beta/tutorials/eager/custom_training_walkthrough.ipynb]: Italic as above

Comment (cre8tor) [inline site/ko/beta/tutorials/eager/custom_training_walkthrough.ipynb]: 손실 함수를 최소화 -> \`손실\` 함수를 최소화

File: site/ko/beta/tutorials/eager/custom_training_walkthrough.ipynb [hunk 6]
```diff
@@ -0,0 +1,1156 @@
...
+    "  <tr><td>\n",
+    "    <img src=\"https://cs231n.github.io/assets/nn3/opt1.gif\" width=\"70%\"\n",
+    "         alt=\"Optimization algorithms visualized over time in 3D space.\">\n",
+    "  </td></tr>\n",
+    "  <tr><td align=\"center\">\n",
+    "    <b> 그림 3.</b> 3차원 공간에 대한 최적화 알고리즘 시각화.<br/>(Source: <a href=\"http://cs231n.github.io/neural-networks-3/\">Stanford class CS231n</a>, MIT License, Image credit: <a href=\"https://twitter.com/alecrad\">Alec Radford</a>)\n",
+    "  </td></tr>\n",
+    "</table>\n",
+    "\n",
+    "텐서플로는 훈련을 위해 사용 가능한 여러종류의 [최적화 알고리즘](https://www.tensorflow.org/api_guides/python/train)을 가지고 있습니다. 이번 모델에서는 *[확률적 경사 하강법(stochastic gradient descent, SGD)](https://developers.google.com/machine-learning/crash-course/glossary#gradient_descent)* 을 구현한 [tf.train.GradientDescentOptimizer](https://www.tensorflow.org/api_docs/python/tf/train/GradientDescentOptimizer)를 사용하겠습니다. `learning_rate`은 경사하강 과정의 크기를 나타내는 매개변수이며, 더 나은 결과를 위해 조절가능한 *하이퍼파라미터(hyperparameter)* 입니다. "
```
Comment (cre8tor) [inline site/ko/beta/tutorials/eager/custom_training_walkthrough.ipynb]: '확률적 경사하강법' italic problem.

File: site/ko/beta/tutorials/eager/custom_training_walkthrough.ipynb [hunk 7]
```diff
@@ -0,0 +1,1156 @@
...
+    "모든 사항이 갖춰졌으므로 모델을 훈련할 준비가 되었습니다! 훈련 루프는 더 좋은 예측을 위해 데이터셋을 모델로 제공합니다. 다음의 코드 블럭은 아래의 훈련 단계를 작성한 것입니다. \n",
+    "\n",
+    "1. 각 *에포크(epoch)* 반복. 에포크는 데이터셋을 통과시키는 횟수입니다. \n",
+    "2. 에포크 내에서, *특성* (`x`)와 *레이블* (`y`)가 포함된 훈련 `데이터셋`에 있는 샘플을 반복합니다.\n",
+    "3. 샘플의 특성을 사용하여 결과를 예측 하고 레이블과 비교합니다. 예측의 부정확도를 측정하고 모델의 손실과 그래디언트를 계산하기 위해 사용합니다. \n",
+    "4. 모델의 변수를 업데이트하기 위해 `옵티마이저`를 사용합니다. \n",
+    "5. 시각화를 위해 몇가지 값들을 저장합니다.\n",
+    "6. 각 에포크를 반복합니다.\n",
+    "\n",
+    "`num_epochs` 변수는 데이터셋의 반복 횟수입니다. 직관과는 반대로, 모델을 길게 학습하는 것이 더 나은 모델이 될 것이라고 보장하지 못합니다. `num_epochs`는 조정가능한 *[하이퍼파라미터(hyperparameter)](https://developers.google.com/machine-learning/glossary/#hyperparameter)* 입니다. 적절한 횟수를 선택하는 것은 많은 경험과 직관을 필요로 합니다. "
```
Comment (cre8tor) [inline site/ko/beta/tutorials/eager/custom_training_walkthrough.ipynb]: Italic issue as above

File: site/ko/beta/tutorials/eager/custom_training_walkthrough.ipynb [hunk 8]
```diff
@@ -0,0 +1,1156 @@
...
+   "outputs": [],
+   "source": [
+    "fig, axes = plt.subplots(2, sharex=True, figsize=(12, 8))\n",
+    "fig.suptitle('Training Metrics')\n",
+    "\n",
+    "axes[0].set_ylabel(\"Loss\", fontsize=14)\n",
+    "axes[0].plot(train_loss_results)\n",
+    "\n",
+    "axes[1].set_ylabel(\"Accuracy\", fontsize=14)\n",
+    "axes[1].set_xlabel(\"Epoch\", fontsize=14)\n",
```
Comment (cre8tor) [inline site/ko/beta/tutorials/eager/custom_training_walkthrough.ipynb]: 'Training Metrics', 'Loss', 'Accuracy', 'Epoch' need translation for plotting

File: site/ko/beta/tutorials/eager/custom_training_walkthrough.ipynb [hunk 9]
```diff
@@ -0,0 +1,1156 @@
...
+   "metadata": {
+    "colab_type": "text",
+    "id": "Zg8GoMZhLpGH"
+   },
+   "source": [
+    "## 모델 유효성 평가\n",
+    "\n",
+    "이제 모델은 훈련되었습니다. 모델의 성능에 대한 몇가지 통계를 얻을 수 있습니다. \n",
+    "\n",
+    "*평가(Evaluating)*는 모델이 예측을 얼마나 효과적으로 수행하는지 결정하는 것을 의미합니다. 붓꽃 분류 모델의 유효성을 결정하기 위해, 몇가지 꽃잎과 꽃받침 데이터를 통과시키고 어떠한 품종을 예측하는지 확인합니다. 그 후 실제 품종과 비교합니다. 예를 들어, 절반의 데이터를 올바르게 예측한 모델의 *[정확도](https://developers.google.com/machine-learning/glossary/#accuracy)* 는 `0.5`입니다. 그림 4는 조금 더 효과적인 모델입니다. 5개의 예측 중 4개를 올바르게 예측하여 80% 정확도를 냅니다.\n",
```
Comment (cre8tor) [inline site/ko/beta/tutorials/eager/custom_training_walkthrough.ipynb]: italic issue for '정확도'

File: site/ko/beta/tutorials/eager/custom_training_walkthrough.ipynb [hunk 10]
```diff
@@ -0,0 +1,1156 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "z-EvK7hGL0d8"
+   },
+   "source": [
+    "### 테스트 데이터 세트 설정\n",
```
Comment (cre8tor) [inline site/ko/beta/tutorials/eager/custom_training_walkthrough.ipynb]: 데이터 세트 -> 데이터셋 for consistency

File: site/ko/beta/tutorials/eager/custom_training_walkthrough.ipynb [hunk 11]
```diff
@@ -0,0 +1,1156 @@
...
+    "    [6.9, 3.1, 5.4, 2.1]\n",
+    "])\n",
+    "\n",
+    "predictions = model(predict_dataset)\n",
+    "\n",
+    "for i, logits in enumerate(predictions):\n",
+    "  class_idx = tf.argmax(logits).numpy()\n",
+    "  p = tf.nn.softmax(logits)[class_idx]\n",
+    "  name = class_names[class_idx]\n",
+    "  print(\"예 {} 예측: {} ({:4.1f}%)\".format(i, name, 100*p))"
```
Comment (cre8tor) [inline site/ko/beta/tutorials/eager/custom_training_walkthrough.ipynb]: 'Example' is translated '샘플' above.
I suggest '샘플' for consistency

**Comment by Choiuijin1125** [comment]
@cre8tor Thank you for review this PR, I'll figure it out ASAP :) thanks 👍

File: site/ko/beta/tutorials/eager/custom_training_walkthrough.ipynb (hunk 1 above)
Comment (Choiuijin1125) [reply site/ko/beta/tutorials/eager/custom_training_walkthrough.ipynb]: Oh, maybe it about a title. If you add English sentence on the title, it's hard to navigate(e.g  https://www.tensorflow.org/alpha/guide/effective_tf2) :)

File: site/ko/beta/tutorials/eager/automatic_differentiation.ipynb [hunk 12]
```diff
@@ -0,0 +1,348 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "1CLWJl0QliB0"
+   },
+   "source": [
+    "## 그래디언트 테이프\n",
+    "\n",
+    "텐서플로는 자동 미분(주어진 입력 변수에 대한 연산의 그래디언트(gradient)를 계산하는 것)을 위한 [tf.GradientTape](https://www.tensorflow.org/api_docs/python/tf/GradientTape) API를 제공합니다. `tf.GradientTape`는 안에서 실행된 모든 연산을 테이프(tape)에 \"기록\"합니다. 그리고 [후진 방식 자동 미분(reverse mode differentiation)](https://en.wikipedia.org/wiki/Automatic_differentiation)을 사용하여 각각의 기록된 연산과 관련된 그래디언트와 테이프를 사용하여 기록된 연산의 그래디언트를 계산합니다. \n",
```
[the remark this answers is not in the document]
Comment (Choiuijin1125) [reply site/ko/beta/tutorials/eager/automatic_differentiation.ipynb]: The title is used for navigation in tensorflow.org (e.g, https://www.tensorflow.org/alpha/guide/effective_tf2).
So long navigation will not give a good user experience. So I'm trying to use Eng in a body :)

**Comment by Choiuijin1125** [comment]
@cre8tor I solved all of your comment. :) but there is commit issue so I'll close this PR and re-upload New PR
